src/app/i18n.js

```diff
@@ -0,0 +1,49 @@
+import i18n from "i18next";
```

Comment:
How do we deactivate the i18n logs both on server and client side?

Reply:
.init({
    **debug: true,** <- here
    fallbackLng: "fr",
    lng: ["en", "fr"],
    interpolation: {
      escapeValue: false,
    },

Reply:
I would make this an env var and set it to false by default.

Reply:
is it really relevant since we can just put it at false in the i18n file. At the end if we do a complete debug feture that activate multiple debug feature this could be helpfull but as now in my way to see how it work moving it to an env file would only be to move from a place to another.